### PR TITLE
Che#2386: Add default README.md file to .che folder

### PR DESCRIPTION
**Pull request by svor**
### What does this PR do?

Provides ability to add and edit default `README.md` file to a project's `.che` folder. File is located in `${che.home}/readme` folder in che distribution and is mounted to each workspace container on its creation.
On project creation/import opens `README.md` file if it exists otherwise opens a default `README.md` file.
### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/2386

### CODE REVIEW

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/readme/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2012-2016 Codenvy, S.A.
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Eclipse Public License v1.0
+which accompanies this distribution, and is available at
+http://www.eclipse.org/legal/epl-v10.html
+
+Contributors:
+   Codenvy, S.A. - initial API and implementation
+
```
Comment (benoitf) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/readme/README.md:19]: default README shouldn't be more complete ? like explaining some stuff ?

Comment (bmicklea) [reply assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/readme/README.md:19]: I just edited the readme.md to add more info.

**Comment by bmicklea** [comment]
Will push this issue because we're working on more substantial architectural changes that will help with this.

**Review by bmicklea — APPROVED** [review]

**Comment by vparfonov** [comment]
We going to close this PR not good solution for now

**Pull request closed without merging**